components/list/list.js

```diff
@@ -210,6 +212,20 @@ class List extends PageableMixin(SelectionMixin(LitElement)) {
 		return items.length > 0 ?  items[0]._getFlattenedListItems().lazyLoadListItems : new Map();
 	}
 
+	_handleColorChange(value, updateSubscribers) {
+		this._childHasColor = value;
+		if (this._controls) this._controls.childHasColor = this._childHasColor;
+		if (updateSubscribers) this._listChildrenUpdatedSubscribers.updateSubscribers();
+	}
+
+	_handleControlsSlotChange(e) {
```

Comment:
The [ListControls](https://github.com/BrightspaceUI/core/blob/main/components/list/list-controls.js#L8) extends [SelectionControls](https://github.com/BrightspaceUI/core/blob/main/components/selection/selection-controls.js#L18)  which supports [selectionFor](https://github.com/BrightspaceUI/core/blob/main/components/selection/selection-observer-mixin.js#L12) which enables the `d2l-list-controls` to be wired up outside of list's `controls` slot, so I don't think `slotchange` will be enough to cover our bases here. Instead, I _think_ the observer (`d2l-list-controls` in this case) should be updated when the `onSubscribe` callback is invoked, but I could be missing part of how that's supposed to work.